.travis.yml

```diff
@@ -4,7 +4,7 @@ language: python
 
 python:
     - "2.7"
-
+dist: trusty
```

Comment:
this was necessary to address a somewhat randomly occurring travis problem. https://openscience.atlassian.net/browse/ENG-703

Reply:
This passed travis here: hhttps://travis-ci.org/Johnetordoff/osf.io/builds/560646183 however the OSF branch is still failing a single test.

Reply:
Yes we should sort out why this is failing before this is merged but [as you can see with this PR](https://github.com/CenterForOpenScience/osf.io/pull/9099) the test is likely failing on develop too. However it doesn't fail locally, I don't know in you have Travis permissions, but I'd recommend clearing the cache and restarting and seeing if that resolves it.

Reply:
Ah, thanks, I hadn't cleared any cache but my own before, but turns out I do have perms!  Clearing the cache resolved.